mentoring/tests/integration/test_mcq.py

```diff
@@ -231,3 +258,20 @@ def test_questionnaire_html_choices(self, page):
         self.wait_until_disabled(submit)
 
         self.assertIn('Congratulations!', messages.text)
+
+
+class MCQBlockBlankThemeTest(MCQBlockTest):
+    """
+    Test MRQ/MCQ questions without the LMS theme which is on by default.
+    """
+    def __init__(self, *args, **kwargs):
+        super(MCQBlockBlankThemeTest, self).__init__(*args, **kwargs)
```

Comment:
If all your `__init__()` method is doing is calling the parent, you don't even need to define it to begin with.